accounting/accounting_cli.py

```diff
@@ -201,7 +201,13 @@ def main():
         elif args.func == "edit_user":
             aclif.edit_user(conn, args.username, args.field, args.new_value)
         elif args.func == "view_job_records":
-            aclif.view_job_records(conn, args.output_file, args)
+            options = {
+                "jobid": args.jobid,
+                "user": args.user,
+                "after_start_time": args.after_start_time,
+                "before_end_time": args.before_end_time,
+            }
+            aclif.view_job_records(conn, output_file, **options)
```

Comment:
I think this is fine, but I think you can also do:

```
aclif.view_job_records(conn, output_file,  jobid=args.jobid, user=args.user,...)
```

not sure if that would be easier to read?  That's just a personal style preference.  I'll approve but leave it to you to decide how best to code up.

Reply:
Thanks @chu11! I'd agree with you - I think this looks a little nicer to read. I pushed a change to use this format instead.